pkg/gencosmosdb/cosmosdb/cosmosdb.go

```diff
@@ -110,7 +113,16 @@ func (c *databaseClient) do(ctx context.Context, method, path, resourceType, res
 }
 
 func (c *databaseClient) _do(ctx context.Context, method, path, resourceType, resourceLink string, expectedStatusCode int, in, out interface{}, headers http.Header) (*http.Response, error) {
-	req, err := http.NewRequestWithContext(ctx, method, "https://"+c.databaseAccount+".documents.azure.com/"+path, nil)
```

Comment:
how about let's change `databaseAccount` to `databaseHostname` and do all the environment logic outside the library?